src/commands/env/resume/sandbox.ts

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { Flags } from '@salesforce/sf-plugins-core';
+import {
+  GlobalInfo,
+  Lifecycle,
+  Messages,
+  Org,
+  ResultEvent,
+  SandboxEvents,
+  SandboxProcessObject,
+  ResumeSandboxRequest,
+  SandboxUserAuthResponse,
+} from '@salesforce/core';
+import { Duration } from '@salesforce/kit';
+import * as Interfaces from '@oclif/core/lib/interfaces';
+import { SandboxCommandBase } from '../../../shared/sandboxCommandBase';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-env', 'resume.sandbox');
+
+type FlagsDef = {
+  'set-default': boolean;
+  alias: string;
+  'poll-interval': Duration;
+  wait: Duration;
+  json: boolean;
+  name: string;
+  'job-id': string;
+  'target-org': Org;
+  'use-most-recent': boolean;
+};
+
+export default class ResumeSandbox extends SandboxCommandBase<SandboxProcessObject> {
+  public static summary = messages.getMessage('summary');
+  public static description = messages.getMessage('description');
+  public static examples = messages.getMessages('examples');
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  public static flags: Interfaces.FlagInput<any> = {
+    'set-default': Flags.boolean({
+      char: 's',
+      summary: messages.getMessage('flags.setDefault.summary'),
+    }),
+    alias: Flags.string({
+      char: 'a',
+      summary: messages.getMessage('flags.alias.summary'),
+      description: messages.getMessage('flags.alias.description'),
+    }),
+    wait: Flags.duration({
+      char: 'w',
+      summary: messages.getMessage('flags.wait.summary'),
+      min: 0,
+      unit: 'minutes',
+      defaultValue: 0,
+    }),
+    'poll-interval': Flags.duration({
```

Comment:
does this add a lot of value?  Asking cuz
1. the other commands don't have it
2. it's adding complexity (see below interactions with wait time)

Reply:
Given sandboxes take so long to complete, being able to control how often a status check is made gives customer ability to control API limits. I can see that this may be obsolete, given resume. Thoughts @mshanemc